core/timeline_inc.php

```diff
@@ -19,17 +19,36 @@
 
 define( 'MAX_EVENTS', 50 );
 
+# Variables that are defined in parent script:
+#
+# $inc_timeline_filter	Filter array to be used to get timeline event
+#						If undefined, it's initialized as null.
+# $inc_timeline_user		User id to limit timeline scope.
+#						If undefined, it's initialized as null.
+#
+
+if( !isset( $inc_timeline_filter ) ) {
```

Comment:
At the moment I would suggest the following:

- Use `$g_` for all global variables.  If we want to limit this to just configuration, then we have a lot of existing code to revisit anyway.  We currently use that for API specific caches, etc.
- Use `$t_` for variables that are only used within a php function or a single php file.  Use of `$t_` for a variable that spans multiple files is not right and should be fixed over time.
- We could revisit the standards at one point, but for now, let's stick with what we have.

> Maybe the comment block at the start of the included script is enough, and keep them as $g_?

Yes.

> Note that we have, in other included scripts, variables with prefix $t_, which is a confusing situation. Having experience that, i tried to make it clear here.

We should fix this over time.